src/Artsy/Router/Utils/RenderStatus.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import StaticContainer from "react-static-container"
+
+import { Box } from "@artsy/palette"
+import ElementsRenderer from "found/lib/ElementsRenderer"
+
+export const RenderPending: React.FC = props => {
+  /**
+   * TODO: Add timeout here for when a request takes too long. Show generic error
+   * and notify Sentry.
+   */
+  return (
+    <Box>
+      <StaticContainer>{null}</StaticContainer>
```

Comment:
So this is just used to stop reconciliation? It seems like the relationship between `Box` and `StaticContainer` should always be the same between `RenderPending` and `RenderReady`. Is that the case?

Reply:
Can you elaborate?

Reply:
If someone came along and added an element between `<Box>` and `<StaticContainer>` in one of these (but not in the other) would it break?

Reply:
@zephraph - yup, impressive eye! i can move this into a shared component